### PR TITLE
Set fixed date for `now` game date in MatchTestData

### PR DESCRIPTION
**Pull request by TBonnin**
## What does this change?

tl;dr: Don't make tests rely on current datetime. :)

One FixturesList feature consists in displaying a list of game matching
some criteria. The list is restricted to the eligible games scheduled
within 3 days of the earliest game.
In MatchTestData, mocked games are created with a relative date base on
`now`.
The earliest game date used to be now+50mins
Therefore if running the tests after 23:10, the earliest game would have
a date set to 'tomorrow'. FixturesList will list games for 3 days
starting tomorrow and not today. Tests fails then since the games in the
list are not the one expected.
Example:
It is Aug 4th at 23:30. First game is set to Aug 5th at 00:20.
FixturesList then returned all the games schedule on Aug 5th, 6th and
7th. However the test is expecting games from Aug 4th (now) to Aug 6th

The simple fix is to set a fixed date for `val now` so tests don't rely
on current datetime and are perfectly repeatable
## What is the value of this and can you measure success?

Tests don't fail if run between 23:10 and 00:00
## Does this affect other platforms - Amp, Apps, etc?

No
## Request for comment

@johnduffell @jfsoul @markjamesbutler

### CODE REVIEW

**Comment by johnduffell** [comment]
perhaps truncate the `now` to a date without time (midnight) and then add on 15 hours? but either way, 👍

**Comment by TBonnin** [comment]
The tests assume now is when the test are running. So changing the meaning of now in the test data will change the expected result

**Comment by johnduffell** [comment]
relying on the current date in test is always bad for repeatability due to non determinism, I suppose my preference would be to freeze it in time

**Comment by TBonnin** [comment]
Yep you are right. Will do that tomorrow

**Comment by TBonnin** [comment]
@johnduffell 574b821 👯

**Comment by johnduffell** [comment]
great! 👍
